docs/deploying-to-production.md

```diff
@@ -0,0 +1,121 @@
+# Deploying to production
+
+## Getting started
+
+Your production enviroment needs to have [Node.js](https://nodejs.org/) and
+[Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/) installed.
+
+After this you need to do the following steps:
+
+1.  [Set environment variables](#enviroment-variables)
+2.  [Build the app](#building-the-app)
+3.  [Run the node server](#starting-the-app)
+
+### Enviroment variables
+
+Before deploying the application make sure you know at least the following variables:
```

Comment:
~~Before deploying the application make sure you know~~  To deploy the application add at least the following variables: